src/scrabble.js

```diff
@@ -1,5 +1,77 @@
-function scrabble() {
-  // write code here
+// the module.exports seems to be working fine
+const { findLettersInCurlyBrackets } = require('./scoreModifiers.js')
+const { findLettersInSquareBrackets } = require('./scoreModifiers.js')
+const { doubleScore } = require('./scoreModifiers.js')
+const { tripleScore } = require('./scoreModifiers.js')
+let wordScore = 0
+
+// might want to store those in an object
+const lettersWorthOne = ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't']
+const lettersWorthTwo = ['d', 'g']
+const lettersWorthThree = ['b', 'c', 'm', 'p']
+const lettersWorthFour = ['f', 'h', 'v', 'w', 'y']
+const lettersWorthFive = ['k']
+const lettersWorthEight = ['j', 'x']
+const lettersWorthTen = ['q', 'z']
```

Comment:
Yes, could perhaps use an object.

Reply:
done! It does look more tidy that way, if anything